docs/pages/api/rbac.mdx

```diff
@@ -0,0 +1,984 @@
+proxy requests to the Kubernetes cluster.
+
+Run the following command, ensuring that <Var name="proxy-address" /> includes
+the host and port of your Proxy Service:
+
+```code
+$ tctl auth sign --user=sync-kubernetes-rbac \
+  --kube-cluster-name=minikube \
+  --format=kubernetes \
+  --proxy=https://<Var name="proxy-address" /> \
+  --out=kubeconfig
+```
+
+### Imports
+
+Here are the packages our client application imports from Go's standard library:
+
+|Package|Description|
+|---|---|
+|`context`|Includes the `context.Context` type. `context.Context` is an abstraction for controlling long-running routines, such as connections to external services, that might fail or time out. Programs can cancel contexts or assign them timeouts and metadata. |
+|`fmt`|Formatting data for printing, strings, or errors.|
+|`io`|Dealing with I/O operations, e.g., reading files or network sockets.|
+|`os`|Interacting with the operating system, e.g., to open files.|
+|`time`|Dealing with time. We will use this to define a timeout for connecting to the Auth Service along with a ticker for executing our discovery logic in a loop.|
+
+The client imports the following third-party code:
+
+|Package|Description|
+|---|---|
+|`github.com/gravitational/teleport/api/client`|A library for authenticating to the Auth Service's gRPC API and making requests, aliased as `teleport`.|
+|`github.com/gravitational/teleport/api/types`|Types used in the Auth Service API, e.g., Application Service records.|
+|`github.com/gravitational/trace`|Presenting errors with more useful detail than the standard library provides.|
+|`google.golang.org/grpc`|The gRPC client and server library.|
+|`k8s.io/api/rbac/v1`|The Kubernetes RBAC API client library.|
+|`k8s.io/apimachinery/pkg/apis/meta/v1`|Code common to Kubernetes' API client libraries.|
+|`k8s.io/client-go/kubernetes`|Setting up an general-purpose Kubernetes client.|
+|`k8s.io/client-go/kubernetes/typed/rbac/v1`|Types for the Kubernetes RBAC API.|
+|`k8s.io/client-go/tools/clientcmd`|Another general-purpose Kubernetes client library.|
+
+### Constants
+
+The program defines constants in a visible location so, later on, it's easier to
+make them configurable outside the program.
+
+Add the following to `main.go` after the package imports:
+
+```go
+const (
+	proxyAddr         string = ""
+	initTimeout              = time.Duration(30) * time.Second
+	identityFilePath  string = "auth.pem"
+	kubeconfigPath    string = "kubeconfig"
+	clusterName       string = "minikube"
+	roleAnnotationKey string = "create-teleport-role"
+)
+```
+
+We will use these constants later in the program. They define some values we may
+want to change later, including:
+
+|Constant|Description|
+|---|---|
+|`proxyAddr`|The host and port of the Teleport Proxy Service, e.g., `mytenant.teleport.sh:443`, which we will use to connect the client to your cluster. **Assign this to your own Proxy Service's host and port.**|
```

Comment:
I'm not sure what you'd do with that, since there's nothing you would copy/pate beyond the variable value you just entered.

Reply:
Right, but we've used it above, so it would already be filled out here below. This way the reader would just get the entered value to use and we wouldn't have to specify an example.